e2e-polybft/framework/test-cluster.go

```diff
@@ -680,7 +680,6 @@ func (c *TestCluster) InitTestServer(t *testing.T,
 
 	srv := NewTestServer(t, c.Config, bridgeJSONRPC, func(config *TestServerConfig) {
 		config.DataDir = dataDir
-		config.Validator = nodeType.IsSet(Validator)
```

Comment:
Should we leave this line?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/2024/commits/e172f51883050e51522a9253a9c313eaaac6eb24 (although it is just being set, and not checked anywhere at the moment).